$OpenBSD: patch-src_backends_lua_luahelper_cpp,v 1.1 2017/11/20 15:44:30 rsadowski Exp $

commit 0b6f7a5721e779dc6f67bc22d5b2511d010f19da
Author: Vadim A. Misbakh-Soloviov <git@mva.name>
Date:   Wed, 15 Feb 2017 15:21:47 -0300

Fix the hardcoded include for lua backend

Currently Cantor has a hardcoded include to get lua.hpp library.
This patch fix the search to include different versions of lua
library. The hardcoded solution is no more necessary.

Index: src/backends/lua/luahelper.cpp
--- src/backends/lua/luahelper.cpp.orig
+++ src/backends/lua/luahelper.cpp
@@ -20,7 +20,7 @@
 
 #include "luahelper.h"
 
-#include <luajit-2.0/lua.hpp> // need the luajit-2.0 prefix to avoid conflicts with Lua 5.2
+#include <lua.hpp>
 #include <QString>
 #include <QStringList>
 #include <kdebug.h>
